qiskit_nature/second_q/problems/base_problem.py

```diff
@@ -20,7 +20,9 @@
 import numpy as np
 from qiskit.algorithms.eigensolvers import EigensolverResult
 from qiskit.algorithms.minimum_eigensolvers import MinimumEigensolverResult
-from qiskit.opflow import Z2Symmetries
+from qiskit.opflow.primitive_ops import Z2Symmetries
+from qiskit.quantum_info.analysis.z2_symmetries import Z2Symmetries as Z2SparseSymmetries
```

Comment:
```suggestion
from qiskit.opflow.primitive_ops import Z2Symmetries as OpflowZ2Symmetries
from qiskit.quantum_info.analysis.z2_symmetries import Z2Symmetries
```
I would rather rename the import which is supposed to go away in the future.
Could you please update this accordingly elsewhere, too?

Reply:
sure